moc-health-check/test_setup.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+
+import sys, os
+sys.path.append(os.path.abspath('../'))
+import subprocess, os
+from subprocess import Popen, PIPE
+import shlex
+
+"""
+Creates "test" directories of varying sizes on which the pman and pfioh tests will be performed
+"""
+
+def getsize(thisDir):
+    total = 0
+    for x in os.listdir(thisDir):
+        dirpath = '%s/%s' % (thisDir, x)
+        fsize = os.path.getsize(dirpath)
+        total += fsize
+
+    total /= (1024 * 1024)
+    return total
+
+def check():
+
+    if not (os.path.exists('/tmp/share')):
+        subprocess.call('mkdir /tmp/share', shell=True)
+
+    if not (os.path.exists('/tmp/small') and os.path.exists('/tmp/medium') and os.path.exists('/tmp/large') and os.path.exists('/tmp/xlarge')):
+        subprocess.call('mkdir /tmp/xlarge', shell=True)
+        subprocess.call('mkdir /tmp/large', shell=True)
+        subprocess.call('mkdir /tmp/medium', shell=True)
+        subprocess.call('mkdir /tmp/small', shell=True)
+
+    if not ((getsize('/tmp/small') == 1) and (getsize('/tmp/medium') == 25) and (getsize('/tmp/large') == 100)):
+            create()
+
+def create():
+    """                                                                                                                                                   
+    Create directories (small, medium, large)                                                                                                            
+    """
+
+    count = 0
+
+    # Fill SMALL directory: should sum to 1MB

```

Comment:
I think we can remove all these options and just take a filesize in mb as the 1 option

Reply:
This method offers more flexibility to testers since they can choose to test the degree of reliability that ChRIS framework provides. They can choose any size of files as per their requirements. So I think this option would be a good addition to the code

Reply:
I am suggesting let's change the model to instead take a param of what size file to use.  Rather than having 4 hardcoded arbitrary sizes.